packages/odyssey-storybook/test/testcafe/framework/pageobjects/buttonPageObject.ts

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {baseUrl} from "../mocks/baseUrl";
+import XPathSelector from "../xpath-selector";
+
+const id = 'components-button'
+const viewMode = 'story'
+const buttonXPathSelector = '//div[@id=\'root\']//Button';
```

Comment:
Pretty sure XPath selection completely bypasses Testing Library.

Reply:
@KevinGhadyani-Okta TestingLibrary isn't being used for the TestCafe tests here!